xyz

```diff
@@ -119,8 +119,14 @@ next_version=$("$dir/node_modules/.bin/semver" -i "$increment" "$version") ||
 message="${message_template//X.Y.Z/$next_version}"
 tag="${tag_template//X.Y.Z/$next_version}"
 
-bold=$(tput bold)
-reset=$(tput sgr0)
+if [[ -t 1 ]] && [[ -n "$TERM" ]] && which tput &>/dev/null; then
```

Comment:
Is this to ensure that stdout is a terminal rather than a file or something else? It seems we could safely assume this since `xyz` is intended to be used interactively.